app/helpers/job_invocation_output_helper.rb

```diff
@@ -0,0 +1,33 @@
+module JobInvocationOutputHelper
+
+  def colorize_line(line)
```

Comment:
afair there is a client side (javascript) lib that convert terminal to html colors.